public/refs/components/schemas/Video.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "description": "The video transcode",
+  "properties": {
+    "url": {
+      "type": "string",
+      "nullable": true,
+      "description": "The asset  video url",
+      "example": "http://path-to-video"
```

Comment:
don't you want to use the same url as in `Asset.json` schema, but with mp4 extension?